src/Native/build-native.sh

```diff
@@ -243,18 +248,29 @@ while :; do
         arm64)
             __BuildArch=arm64
             ;;
-        debug)
+        debug|-debug)
             __BuildType=Debug
             ;;
-        release)
+        release|-release)
             __BuildType=Release
             __CMakeArgs=RELEASE
             ;;
+        -osgroup*)
+             __BuildOS=`echo $1 | cut -d '=' -f2`
+             __OSGroupSet=1
+             ;;
+        --osgroup*)
```

Comment:
Can you please unify the two option variants like you did for coreclr?

Reply:
I could but here is the catch. Unlike coreclr to the top level make is managed. To build for FreeBSD I use build.sh -OSGroup=FreeBSD
That calls native build with run.exe as 
/home/furt/git/wfurt2-corefx/src/Native/build-native.sh  --OSGroup=FreeBSD  x64  Debug  Linux  --numproc 4  -portable  toolSetDir=c:\tools\clr

maybe @weshaggard or @eerhardt have some better suggestions but I want to set BuildOS to FreeBSD but leave HostOS to Linux. 

The OSGroup also works only in parts or managed code. It creates tests in proper directories but for example RunTests.sh still has references to linux e.g. HostOS.

Reply:
I am not sure I understand. I was just suggesting to unify the `--osgroup` and `-osgroup` options so that the case is `--osgroup*|-osgroup*)` since their bodies are the same.

Reply:
ok. that makes sense. thanks @janvorli